hook.js

```diff
@@ -9,68 +10,75 @@ var parseSource = require('./sourceParse');
 
 var token = 'token ' + process.env.OAUTH_TOKEN;
 var productName = 'Discord';
+var githubClient = github.client();
 
-var comment = function(request, response) {
-    var payload = request.body;
-    var configMetadataURL = payload.repository.contents_url.replace('{+path}', '.doiuse');
+function handle(request, response) {
+    var eventType = request.headers['x-github-event'];
+    var metadata = request.body;
 
+    // React to pull requests only
+    if(eventType === 'pull_request') {
+       response.status(200).send('OK');
```

Comment:
I was also wondering about this. If someone sends us another event, like an issue comment event, do we want to respond `200` to that? I guess I'm not sure what response code would be more appropriate.